data/projects/all.json

```diff
@@ -235,9 +235,52 @@
             "sonatype_lifecycle": false
         },
         "tags": [
-            "nexus",
-            "repository",
             "blobstore"
         ]
+    },
+    {
+        "name": "API Clients",
+        "image": "github-mark.png",
+        "description": "Generated API Clients in Go, Java and Typescript for Sonatype Nexus Repository Manager",
+        "active": true,
+        "flagship": false,
+        "graduated": false,
+        "repository": "nexus-repo-api-client",
+        "works_with": {
+            "ossindex": false,
+            "sonatype_nexus_repository": true,
+            "sonatype_lifecycle": false
+        },
+        "tags": [
+            "golang",
+            "java",
+            "typescript",
+            "api",
+            "client"
+        ]
+    },
+    {
+        "name": "API Clients",
```

Comment:
```suggestion
        "name": "IQ Server API Clients",
```

Reply:
Not aligned to brand - should be at least `Sonatype IQ Server...`

Reply:
"API Clients (IQ)"

Reply:
This PR is merged - new PR?

Reply:
https://github.com/sonatype-nexus-community/contribute.sonatype.com/pull/42